man/systemd.kill.xml

```diff
@@ -94,7 +94,8 @@
         enabled with <varname>SendSIGHUP=</varname>). If then, after a
         delay (configured via the <varname>TimeoutStopSec=</varname>
         option), processes still remain, the termination request is
-        repeated with the <constant>SIGKILL</constant> signal (unless
+        repeated with the <constant>SIGKILL</constant> signal or the
+        signal specified via <varname>FinalKillSignal</varname> (unless
```

Comment:
we alway suffix settings with "=" in the docs, please do so here too. That way it we hint to the reader that this is a unit file setting